### PR TITLE
cask move_back: copy / delete

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----
Fixes https://github.com/caskroom/homebrew-cask/issues/42112 ~~(maybe)~~

cc @reitermarkus

### CODE REVIEW

**Comment by commitay** [comment]
```
2.00s==> brew config
HOMEBREW_VERSION: 1.5.2-5-g82fa44f
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 82fa44f060eaa344f0a4e0dd7d84d20d90e5be12
Last commit: 26 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 031b88db283123420d0b7f5c22e5655593897544
Core tap last commit: 4 months ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /Users/travis/build/Homebrew/brew
HOMEBREW_DEVELOPER: 1
HOMEBREW_ENV_FILTERING: 1
HOMEBREW_FAIL_LOG_LINES: 150
HOMEBREW_FORCE_VENDOR_RUBY: 1
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_NO_ANALYTICS_THIS_RUN: 1
HOMEBREW_NO_AUTO_UPDATE: 1
HOMEBREW_NO_EMOJI: 1
HOMEBREW_REPOSITORY: /Users/travis/build/Homebrew/brew
HOMEBREW_SANDBOX: 1
HOMEBREW_VERBOSE_USING_DOTS: 1
CPU: dual-core 64-bit ivybridge
Homebrew Ruby: 2.3.3 => /Users/travis/build/Homebrew/brew/Library/Homebrew/vendor/portable-ruby/2.3.3/bin/ruby
Clang: 9.0 build 900
Git: 2.13.5 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Error: Calling <<-EOS.undent is deprecated!
Use <<~EOS instead.
/Users/travis/build/Homebrew/brew/Library/Taps/homebrew/homebrew-core/Formula/python.rb:89:in `block in <class:Python>'
Please report this to the homebrew/core tap!
Or, even better, submit a PR to fix it!
Curl: 7.54.0 => /usr/bin/curl
Perl: /usr/bin/perl
==> FAILED
```

**Comment by commitay** [comment]
@reitermarkus ping

**Comment by reitermarkus** [comment]
> (~~maybe~~)

What caused the change from maybe to ~~maybe~~? Is this reproducible now?

File: Library/Homebrew/cask/lib/hbc/artifact/moved.rb [hunk 1]
```diff
@@ -62,14 +62,16 @@ def move_back(skip: false, force: false, command: nil, **options)
           raise CaskError, "It seems the #{self.class.english_name} source '#{target}' is not there."
         end
 
-        ohai "Moving #{self.class.english_name} '#{target.basename}' back to '#{source}'."
+        ohai "Copying #{self.class.english_name} '#{target.basename}' back to '#{source}'."
```
Comment (reitermarkus) [inline Library/Homebrew/cask/lib/hbc/artifact/moved.rb]: Maybe change this to “Backing … up to …”.

**Comment by commitay** [comment]
> What caused the change from maybe to ~~maybe~~? Is this reproducible now?

I couldn't reliably reproduce the original issue. When this was first opened I wasn't 100% sure it would fix it but I've been using it locally since and haven't had any issues.

File: Library/Homebrew/cask/lib/hbc/artifact/moved.rb (hunk 1 above)
Comment (commitay) [reply Library/Homebrew/cask/lib/hbc/artifact/moved.rb]: Updated.

**Comment by MikeMcQuaid** [comment]
@commitay You can now self-merge this if you're confident. Nice work!

**Comment by commitay** [comment]
Thanks @MikeMcQuaid!

**Pull request merged**